src/components/swaptab/index.js

```diff
@@ -41,18 +41,26 @@ const styles = theme => ({
     backgroundColor: theme.colors.matisseBlue,
     height: '15%',
     width: '100%',
+    justifyContent: 'center',
     alignItems: 'center',
     '&:hover': {
       cursor: 'pointer',
     },
   },
+  controls: {
+    flex: 2,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
   nextText: {
-    flex: 1,
-    textAlign: 'center',
-    fontSize: '30px',
     color: theme.colors.white,
+    fontWeight: '300',
   },
-  icon: {
+  nextError: {
+    color: theme.colors.lightGrey,
```

Comment:
I would rather make the background of the button grey instead of the font. I could barely notice a difference when the color of the font changed